packages/web-components/src/components/expressive-modal/__stories__/README.stories.mdx

```diff
@@ -45,19 +45,19 @@ or with [JSPM](https://jspm.org)
 ### HTML
 
 ```html
-<dds-modal open>
-  <dds-modal-header>
-    <dds-modal-close-button role="button"></dds-modal-close-button>
-    <dds-modal-heading>Modal Title</dds-modal-heading>
-  </dds-modal-header>
-  <dds-modal-body>
+<dds-expressive-modal open>
```

Comment:
The JS documentation above I think also needs to change right? the filenames and folders are still the old ones.

Reply:
Great catch @jeffchew - Fixed.